.github/workflows/go-test.yml

```diff
@@ -28,6 +28,14 @@ jobs:
         run: |
           go version
           go env
+      - name: Install FoundationDB Client C bindings
+        env:
+          FDB_VERSION: '7.1.33'
+          FDB_CLIENTS_DEB_SHA256_SUM: 'd73002bc796de7ce0158f1eb64fab1f8e90bada2edd90599d9b40754220150af'
+        run: |
+          wget "https://github.com/apple/foundationdb/releases/download/${FDB_VERSION}/foundationdb-clients_${FDB_VERSION}-1_amd64.deb"
+          echo "${FDB_CLIENTS_DEB_SHA256_SUM}  foundationdb-clients_${FDB_VERSION}-1_amd64.deb" | sha256sum --check
+          sudo dpkg --force-all -i foundationdb-clients_${FDB_VERSION}-1_amd64.deb
       - name: Use msys2 on windows
         if: matrix.os == 'windows'
```

Comment:
I wanted to refrain from modifying autogenerated CI builds as much as I could. Piotr has kindly given me pointer on how to avoid this. 

If sorting out the CI build on mac and windows is a non blocker I'd like to push ahead with testing this on dev before spending more time on FDB.